### ISSUE TITLE
when automated issue reporting from UI leads to 2 different queries

### ISSUE THREAD

**Issue by sandrine-muller-research** [NCATSTranslator/Feedback#768]
This issue has been reported internally within MolePro team by @codewarrior2000 where we have issues where the metadata info reported by UI does not match the same query:
![image](https://github.com/NCATSTranslator/Feedback/assets/22616936/30d3b4c6-8559-4ea9-b38b-830116edf55b)

The first PK reported within the UI URL is about this query:
![image](https://github.com/NCATSTranslator/Feedback/assets/22616936/5d021841-05f1-4a76-9d6d-43d105680768)

While the second PK (from ARS) is about this query:
![image](https://github.com/NCATSTranslator/Feedback/assets/22616936/c9ab938d-6360-47b7-a2c8-f8d420307b89)

It happens that I was running those 2 queries in (almost in parallel, I believe potassium happened first) when submitting the tickets so I wonder whether the mixing could happen from there? 

perhaps not a priority, but it could become an issue if we move forward with the automated parsing of the issues and come up with conflicting queries...

**Comment by gprice1129** [comment]
@sandrine-muller-research we are unable to reproduce this issue. If this is still happening can you give exact reproduction steps? We are going to close the ticket for now.

**Comment by codewarrior2000** [comment]
This issue occurred again today with #[974](https://github.com/NCATSTranslator/Feedback/issues/974) , where the stated ARS PK does not match the URL.

**Comment by dnsmith124** [comment]
Thanks for reopening this @codewarrior2000, taking a look
